src/sentry/models/actor.py

```diff
@@ -148,6 +157,12 @@ def resolve(self) -> Union["Team", "RpcUser"]:
         return fetch_actor_by_id(self.type, self.id)
 
     def resolve_to_actor(self) -> Actor:
+        obj = self.resolve()
+        if obj.actor_id is None and isinstance(obj, RpcUser):  # This can happen for users now.
+            Actor.objects.create(
```

Comment:
Interesting, JIT.

Reply:
This is key to this refactor. I need to make sure we find all the places we need this. Hopefully CI is on my side

Reply:
To be clear, again,
if all you do is start with dual writes without changing reading paths, there is no downside to being wrong.

I think the path forward can be to start making dual writes, then evaluate in production if it closes the loop with tools.

Reply:
Well, because of how user actors are created with signals, it's also not easily possible to assert that I've identified all the places where we would need this if I keep that initial creation point.

Reply:
Yeah that's totally fair.
Curious -- right now, how many users do not have actor_ids?  I'm willing to bet there already exists some bugs related to this.